go/stellar/stellarsvc/frontend.go

```diff
@@ -1229,7 +1236,7 @@ func (s *Server) FindPaymentPathLocal(ctx context.Context, arg stellar1.FindPaym
 		return stellar1.PaymentPathLocal{}, err
 	}
 	srcAmt.Quo(srcAmt, destAmt)
-	res.ExchangeRate = fmt.Sprintf("1 %s = %s %s", path.DestinationAsset.Code, srcAmt.FloatString(7), path.SourceAsset.Code)
+	res.ExchangeRate = fmt.Sprintf("1 %s = %s %s", assetToSuffix(path.DestinationAsset), srcAmt.FloatString(7), assetToSuffix(path.SourceAsset))
```

Comment:
cc @patrickxb

Reply:
we should probably use `stellar.FormatAmountDescriptionAsset()` here.

Reply:
addressed in https://github.com/keybase/client/pull/18169

marking as resolved